README.md

```diff
@@ -603,72 +567,107 @@ a similar concept (variable reference immutability) while this proposal intends
 While immutability is the common thread between these two features, there has been significant community feedback that
 indicates that using `const` in both contexts is undesirable.
 
-Instead of using a keyword, `{| |}` and `[||]` have been suggested as possible alternatives. However, the champion group has strong preference for `#{}` and `#[]` over that alternative.
+Instead of using a keyword, `{| |}` and `[||]` have been suggested as possible alternatives. Currently, the champion group is leaning towards `#[]`/`#{}`, but discussion is ongoing in [#10](https://github.com/tc39/proposal-record-tuple/issues/10).
```

Comment:
Maybe we should also talk about `{| }` and `[| ]`

Reply:
I guess we'll have more to share once we decide on an approach for next steps on this issue, so let's update the README as part of that. cc @getify